File: javascript-sources/chapter-2/demo-01/create-queue.js
const AWS = require('aws-sdk');

AWS.config.region = 'us-west-2';
AWS.config.credentials = new AWS.SharedIniFileCredentials({profile: 'sqs_dev'});

let sqs = new AWS.SQS();

var params = {
  QueueName: 'programmatic-queue-1',
  Attributes: {
    'DelaySeconds': '5'
  }
};
sqs.createQueue(params, function(err, data) {
  if (err)
    console.log(err, err.stack);
  else
    console.log(data);
});

File: python-sources/chapter-2/demo-01/create_queue.py
import boto3

session = boto3.Session(profile_name='sqs_dev')

# Get the service resource
sqs = session.resource('sqs', region_name='us-west-2')

# Create the queue. This returns an SQS.Queue instance
queue = sqs.create_queue(QueueName='programmatic-queue-2', Attributes={'DelaySeconds': '5'})

# You can now access identifiers and attributes
print(queue.url)
print(queue.attributes.get('DelaySeconds'))